recipe/build.sh

```diff
@@ -21,9 +22,20 @@ cd ${SRC_DIR}/lib
 rm cspice.a
 rm csupport.a
 #  compile c code
-${CC} -Iinclude -c -fPIC -m64 -O2 -ansi -pedantic ./../src/cspice/*.c 
+if [ "$CI" == "drone" ];
+then
+    ${CC} -Iinclude -c -fPIC -O2 -ansi -pedantic ./../src/cspice/*.c 
+else
+    ${CC} -Iinclude -c -fPIC -O2 -ansi -pedantic -m64 ./../src/cspice/*.c 
+fi  
```

Comment:
```suggestion
    ${CC} ${CFLAGS} -Iinclude -c -fPIC -O2 -ansi -pedantic ./../src/cspice/*.c 
```

Reply:
ahh didn't think about the cflags, thanks!